pylearn2/datasets/dataset.py

```diff
@@ -2,21 +2,25 @@
 A module defining the Dataset class.
 """
 
+import abc
+import warnings
+from pylearn2.utils.iteration import resolve_iterator_class
+
 
 class Dataset(object):
 
     """
     Abstract interface for Datasets.
     """
+    #__metaclass__ = abc.ABCMeta
```

Comment:
If assigning a metaclass does not work yet, we should remove that comment, and not import `abc`.